pkg/controllers/kubectl_delivery/controller.go

```diff
@@ -31,6 +31,7 @@ import (
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/klog"
+	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
```

Comment:
Can we copy the util from the upstream instead of importing kubernetes?

Reply:
Thanks for your advising! As you mentioned,  I have customized a function to implement same thing instead of importing k8s.io/kuberneters pkg. Could you please check this again?